db/blip.go

```diff
@@ -73,15 +73,20 @@ func defaultBlipLogger(ctx context.Context) blip.LogFn {
 }
 
 // blipRevMessageProperties returns a set of BLIP message properties for the given parameters.
-func blipRevMessageProperties(revisionHistory []string, deleted bool, seq SequenceID) blip.Properties {
+func (bsc *BlipSyncContext) blipRevMessageProperties(revisionHistory []string, deleted bool, seq SequenceID) blip.Properties {
 	properties := make(blip.Properties)
 
 	// TODO: Assert? db.SequenceID.MarshalJSON can never error
 	seqJSON, _ := base.JSONMarshal(seq)
 	properties[RevMessageSequence] = string(seqJSON)
 
 	if len(revisionHistory) > 0 {
-		properties[RevMessageHistory] = strings.Join(revisionHistory, ",")
+		if bsc.activeCBMobileSubprotocol <= CBMobileReplicationV3 {
```

Comment:
It looks like having v4 and above pass in a history of length 1 will already handle this without the additional subprotocol check here (strings.Join already includes streamlined handling for length 0 and length 1 slices).  Given that, do you think we need the additional Subprotocol check here?

Reply:
Yes good point, didn't think of that! Made that change.